packages/palette/src/elements/Input/Input.tsx

```diff
@@ -14,6 +15,7 @@ export interface InputProps
     > {
   active?: boolean
   description?: string
+  descriptionVariant?: TooltipVariant
```

Comment:
I don't think we should support a configurable variant on this. They should all be the same. Safe to remove this prop.